### DISCUSSION TITLE
Docker config changes

### DISCUSSION THREAD

**Discussion by rozatoo** [ORelio/Minecraft-Console-Client#1770] [Q&A] [answered]
How do you edit config files like kickmessages.txt if you've installed the client with Docker?

**Comment by ReinforceZwei** [comment, marked as answer]
I'm not familiar with docker but I think you could mount a volume (i.e. a folder) to a container at `/opt/data` which stores the files, as per in the docker startup script. Then you can modify the configuration files. 

https://docs.docker.com/storage/bind-mounts/

**Comment by Ccode-lang** [comment]
Mabye start up a docker console in the container file and edit the file.
